src/nginxconfig/util/browser_language.js

```diff
@@ -53,7 +50,11 @@ export default () => {
         }, {});
 
         // Try to match a user language to a pack language
-        const langMatch = [...userLocales.values()].find(x => i18nPackLanguages.includes(x.split('-')[0].toLowerCase()));
-        if (langMatch) return i18nPackLanguages[langMatch.split('-')[0].toLowerCase()];
+        if (i18nPackLanguages){
+            const langMatch = [...userLocales.values()].find(x => Object.keys(i18nPackLanguages).includes(x.split('-')[0].toLowerCase()));
+            if (langMatch) return i18nPackLanguages[langMatch.split('-')[0].toLowerCase()];
+        }
```

Comment:
This if statement is not needed, `i18nPackLanguages` will always be truth-y as it generates via a reduce from an empty object.